### PR TITLE
Reformat of American, antelope, Regional Approx, and movement of floodplain files

### PR DESCRIPTION
**Pull request by ErinCain**

### CODE REVIEW

File: data-raw/watershed/regional_approximation/Regional_Approximation.Rmd [hunk 1]
```diff
@@ -16,37 +16,44 @@ library(lubridate)
...
+  Watershed = c("Antelope Creek", "Bear Creek", "Big Chico Creek", "Cow Creek*", 
+                "Elder Creek", "Mill Creek", "Paynes Creek", "Stony Creek", 
+                "Thomes Creek"), 
+  `Area (sqkm)` = c(508, 649, 930, 1107, 358, 334, 243, 2021, 758), 
+  `Width (m)` = c(15.8, 4.3, 14.8, 8.5, 8.6, 14.3, 6.3, 19.6, 12.9)
+) %>% 
+  knitr::kable(align = 'c')
+```
+
+*Cow Creek does not have modeled spawning habitat.
```
Comment (thesadie) [inline data-raw/watershed/regional_approximation/Regional_Approximation.Rmd]: Change to '*Cow Creek has modeled rearing habitat.'

File: data-raw/watershed/regional_approximation/description_regional_approx.R [hunk 2]
```diff
@@ -0,0 +1,18 @@
+library(tidyverse)
+library(readxl)
+library(glue)
+
+print_regional_approx <- function(ws) {
```
Comment (thesadie) [inline data-raw/watershed/regional_approximation/description_regional_approx.R]: change function argument name to `watershed_name` from `ws` and then delete `watershed_name <- ws`

File: data-raw/watershed/floodplain_utils.R [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+  }
+
+  if (str_detect(watershed_method, 'scaled')) {
+    proxies <- c('Deer Creek', 'Cottonwood Creek', 'Tuolumne River')
+    names(proxies) <- c('dc', 'cc', 'tr')
+
+    proxy_ws <- proxies[str_remove(watershed_method, 'scaled_')]
+
+    return(
+      glue("No watershed specific hydraulic modeling was available for {watershed_name}.",
```
Comment (thesadie) [inline data-raw/watershed/floodplain_utils.R]: nit picky comment - single quotes are used throughout this script, you used double. plz make effort to match style of existing code within a script

File: data-raw/watershed/floodplain_utils.R [hunk 4]
```diff
@@ -0,0 +1,237 @@
+library(tidyverse)
```
Comment (thesadie) [inline data-raw/watershed/floodplain_utils.R]: why do you have two versions of this script? `watershed/floodplain_utils.R` and `watershed/floodplain/floodplain_utils.R` I wouldn't nest it a floodplain directory

File: data-raw/watershed/floodplain_utils.R [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+    proxy_ws <- proxies[str_remove(watershed_method, 'scaled_')]
+
+    return(
+      glue("No watershed specific hydraulic modeling was available for {watershed_name}.",
+           " A flow to inundated floodplain area relationship was generated for {watershed_name}",
+           " by scaling the flow to inundated floodplain area relationship for {proxy_ws}.",
+           " This scaling used the ratio of mean flow from December to June between the modeled",
+           " and unmodeled watershed. Flows and corresponding inundated floodplain areas per unit length ",
+           " were calculated for {watershed_name} as {flow_scale}% of {proxy_ws}. ",
+           " Of the entire mapped {rearing_length} mile rearing",
```
Comment (thesadie) [inline data-raw/watershed/floodplain_utils.R]: change 'mile' to 'miles'

File: data-raw/watershed/antelope/antelope_creek.Rmd [hunk 6]
```diff
@@ -16,35 +16,30 @@ library(cvpiaHabitat)
...
 
-## Floodplain Modeling Details
-
-**Data Source:**
+## Floodplain Rearing Habitat 
 
-[Scaled from a Deer Creek flow to floodplain area relationship generated with a 1D HEC-RAS hydraulic model](https://s3-us-west-2.amazonaws.com/cvpiahabitat-r-package/cvpia-sit-model-inputs/DRAFT+Deer+Creek+Hydraulic+Models+Tech+Memo+6-08-07.pdf)
+**Data Source:** [Scaled from a Deer Creek flow to floodplain area relationship generated with a 1D HEC-RAS hydraulic model](https://s3-us-west-2.amazonaws.com/cvpiahabitat-r-package/cvpia-sit-model-inputs/DRAFT+Deer+Creek+Hydraulic+Models+Tech+Memo+6-08-07.pdf)
 
 `r print_model_details('Antelope Creek', 'fr')`
```
Comment (thesadie) [inline data-raw/watershed/antelope/antelope_creek.Rmd]: make sure to rerun this file after fixing the issues i flagged in the `print_model_details`

File: data-raw/watershed/american/american_river.rmd [hunk 7]
```diff
@@ -88,50 +104,59 @@ american_river_instream %>%
...
          ST_floodplain_acres = modeled_floodplain_area_acres) %>%
   select(flow_cfs,
          FR_floodplain_acres = modeled_floodplain_area_acres,
          SR_floodplain_acres, ST_floodplain_acres,
-         watershed)
+         watershed) %>% 
+  filter(flow_cfs >= threshold_flow)
 
 knitr::kable(align = 'c', head(american_river_floodplain, 5))
 # devtools::use_data(american_river_floodplain, overwrite = TRUE)
```
Comment (thesadie) [inline data-raw/watershed/american/american_river.rmd]: replace `devtools` package with `usethis`, add that check to your checklist

File: data-raw/watershed/american/american_river.rmd [hunk 8]
```diff
@@ -88,50 +104,59 @@ american_river_instream %>%
...
          SR_floodplain_acres = modeled_floodplain_area_acres,
          ST_floodplain_acres = modeled_floodplain_area_acres) %>%
   select(flow_cfs,
          FR_floodplain_acres = modeled_floodplain_area_acres,
          SR_floodplain_acres, ST_floodplain_acres,
-         watershed)
+         watershed) %>% 
+  filter(flow_cfs >= threshold_flow)
 
 knitr::kable(align = 'c', head(american_river_floodplain, 5))
```
Comment (thesadie) [inline data-raw/watershed/american/american_river.rmd]: add header descriptions

File: data-raw/watershed/regional_approximation/Regional_Approximation.Rmd (hunk 1 above)
Comment (ErinCain) [reply data-raw/watershed/regional_approximation/Regional_Approximation.Rmd]: done

File: data-raw/watershed/regional_approximation/description_regional_approx.R (hunk 2 above)
Comment (ErinCain) [reply data-raw/watershed/regional_approximation/description_regional_approx.R]: done

File: data-raw/watershed/floodplain_utils.R (hunk 3 above)
Comment (ErinCain) [reply data-raw/watershed/floodplain_utils.R]: done

File: data-raw/watershed/floodplain_utils.R (hunk 4 above)
Comment (ErinCain) [reply data-raw/watershed/floodplain_utils.R]: Un-nested

File: data-raw/watershed/floodplain_utils.R (hunk 5 above)
Comment (ErinCain) [reply data-raw/watershed/floodplain_utils.R]: changed

File: data-raw/watershed/antelope/antelope_creek.Rmd (hunk 6 above)
Comment (ErinCain) [reply data-raw/watershed/antelope/antelope_creek.Rmd]: re-ran

File: data-raw/watershed/american/american_river.rmd (hunk 7 above)
Comment (ErinCain) [reply data-raw/watershed/american/american_river.rmd]: fixed usethis

File: data-raw/watershed/american/american_river.rmd (hunk 8 above)
Comment (ErinCain) [reply data-raw/watershed/american/american_river.rmd]: done

**Pull request merged**